src/app/issue/conversation/conversation.component.ts

```diff
@@ -36,154 +37,111 @@ export class ConversationComponent
     private route: ActivatedRoute,
     private auth: AuthService,
     private issueService: IssueService,
-    private summaryService: IssueSummaryService
+    private summaryService: IssueSummaryService,
+    private projectUserService: ProjectUserService
   ) {
     super();
-    this.auth.currentUser.subscribe((user) => (this.user = user));
   }
 
+  //TODO ForkJoin umbauen
   ngOnInit(): void {
-    this.archivedDisabled = false;
+    this.auth.currentUser.subscribe((user) => (this.user = user));
```

Comment:
Muss hier this.subscribe(...) verwendet werden?

Reply:
Glaube schon. currentUser ist ja ein Observable<User>.